lib/puppet/parser/functions/collectd_convert_processmatch.rb

```diff
@@ -1,4 +1,4 @@
-Puppet::Parser::Functions.newfunction(:collectd_convert_processmatch, :type => :rvalue, :arity => 1, :doc => <<-ENDDOC
+Puppet::Parser::Functions.newfunction(:collectd_convert_processmatch, type: :rvalue, arity: 1, doc: <<-ENDDOC
```

Comment:
It looks as if this is the only change that actually breaks 1.8.7. I guess I'll have to modify locally. :)